src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -338,6 +338,7 @@
     </Compile>
     <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlCommandBuilder.cs">
       <Link>Microsoft\Data\SqlClient\SqlCommandBuilder.cs</Link>
+      <SubType>Component</SubType>
     </Compile>
```

Comment:
```suggestion
      <Link>Microsoft\Data\SqlClient\SqlCommandBuilder.cs</Link>
    </Compile>
```

Reply:
The IDE is just going to keep putting it back until the PR that contains the change in the attribute on the class is merged. Can we just live with it until then?

Reply:
I think you included changes in this [PR](https://github.com/dotnet/SqlClient/pull/1334#issuecomment-998339865) and closed it for pipeline failures. Those changes are missing here.

Reply:
Should be fixed and not come back now.